base/broadcast.jl

```diff
@@ -606,6 +606,7 @@ Base.RefValue{String}("hello")
 ```
 """
 broadcastable(x::Union{Symbol,AbstractString,Function,UndefInitializer,Nothing,RoundingMode,Missing,Val}) = Ref(x)
```

Comment:
Should we consolidate these definitions
```Julia
broadcastable(x::Union{Symbol,AbstractString,Function,UndefInitializer,Nothing,RoundingMode,Missing,Val}) = Ref(x)
broadcastable(x::Enum) = Ref(x)
broadcastable(x::Ptr) = Ref(x)
broadcastable(r::Regex) = Ref(r)
```
as
```Julia
broadcastable(x::Union{Symbol,AbstractString,Function,UndefInitializer,Nothing,RoundingMode,Missing,Val,Ptr,Regex,Enum}) = Ref(x)
```
?

Reply:
The `Enum` definition can't go here, since `Enums.jl` is loaded after this file.   So you need to put `Base.broadcastable(x::Enum) = Ref(x)` into `Enums.jl` instead.

We could additionally consolidate this code somewhat as you say, but for simplicity you could leave it as-is in this PR.